helm/superset/values.yaml

```diff
@@ -278,8 +278,11 @@ supersetNode:
         - /bin/sh
         - -c
         - dockerize -wait "tcp://$DB_HOST:$DB_PORT" -timeout 120s
+
   # -- Annotations to be added to supersetNode deployment
   deploymentAnnotations: {}
+  # -- Labels to be added to supersetNode deployment
+  deploymentLabels: {}
```

Comment:
Pls update the schema to reflect this addition

Reply:
Hi, fixed thanks !